app/views/admin/publishers/edit.html.slim

```diff
@@ -14,6 +14,10 @@
         label
           = f.check_box :excluded_from_payout, {}, checked: @publisher.excluded_from_payout?
           = " Uphold widget disabled"
+          br
```

Comment:
When you click the text it checks the wrong box, so if we change this to be like this it should work.
```suggestion
          br
        label
```